build/DependencyVersions.props

```diff
@@ -1,19 +1,19 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <CLI_SharedFrameworkVersion>2.0.0-beta-001509-00</CLI_SharedFrameworkVersion>
-    <CLI_MSBuild_Version>15.2.0-preview-000002-01</CLI_MSBuild_Version>
-    <CLI_Roslyn_Version>2.0.0-rc4-61325-08</CLI_Roslyn_Version>
-    <CLI_NETSDK_Version>1.1.0-alpha-20170209-1</CLI_NETSDK_Version>
-    <!-- non-official NuGet build taken from https://github.com/nuget/nuget.client/tree/release-4.0.0-rc3-netstandard2.0 to contain "2.0" TFMs -->
-    <CLI_NuGet_Version>4.0.0-rc3-2193</CLI_NuGet_Version>
-    <CLI_WEBSDK_Version>1.0.0-alpha-20170130-3-281</CLI_WEBSDK_Version>
-    <CLI_TestPlatform_Version>15.0.0-preview-20170125-04</CLI_TestPlatform_Version>
-    <SharedFrameworkVersion>$(CLI_SharedFrameworkVersion)</SharedFrameworkVersion>
-    <SharedHostVersion>$(CLI_SharedFrameworkVersion)</SharedHostVersion>
-    <HostFxrVersion>$(CLI_SharedFrameworkVersion)</HostFxrVersion>
-    <TemplateEngineVersion>1.0.0-beta1-20170202-111</TemplateEngineVersion>
-    <TemplateEngineTemplateVersion>1.0.0-beta1-20170131-110</TemplateEngineTemplateVersion>
-    <TemplateEngineTemplate2_0Version>1.0.0-beta1-20170209-117</TemplateEngineTemplate2_0Version>
-  </PropertyGroup>
-</Project>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What editor?  What do you use to merge conflicts?

We should ensure we don't change encodings/line-endings/etc on files because it makes for an absolute nightmare merging.